### ISSUE TITLE
Can't input anything via fcitx in notes on Linux

### ISSUE THREAD

**Issue by hunterforcode** [nuttyartist/notes#180]
Hi, it's a very excellent software for me, however, it's a pity that this software can't input any characters or kanas via fcitx, which is a lightweight input method framework on Linux. 
Here is how to produce:
1 install fcitx and notes software 
2 activate the fcitx ime and try to type something in the notes software.
3 nothing can be typed in the notes except letters. 

I am not sure whether any others who are using fcitx come across the same problem or not. If so, it would be welcomed that more people confirm the bug.

**Comment by theshadowx** [comment]
have you tried other Qt application that use text ?

**Comment by hunterforcode** [comment]
I can input text in most Qt applications. 
To confirm it, would you mind specifying a program to let me try it again?
I follow the instruction here. https://wiki.archlinux.org/index.php/fcitx

**Comment by theshadowx** [comment]
Have installed these packages:

To obtain a better experience in Gtk+ and Qt programs, install the fcitx-gtk2, fcitx-gtk3, fcitx-qt4 and fcitx-qt5 input method modules as your need, or the fcitx-im group to install all of them.

**Comment by hunterforcode** [comment]
Hi, on Ubuntu I have installed similar packages as you say. 
![synaptic screenshot](https://i.imgur.com/GrPzu5z.png)
I can only input letters but still can't input Chinese or Japanese characters and kanas via fcitx.

**Comment by nuttyartist** [comment]
Hi there @hunterforcode, can you try testing if you're having the same problem with this other Qt app http://www.qownnotes.org?

**Comment by hunterforcode** [comment]
Yes, I follow the installation guide [here](http://www.qownnotes.org/installation).
The same problem from my end. 
I also try to use qt5.8, however, the problem still exists.

**Comment by HCY-ASLEEP** [comment]
The problem still exists, can anyone solve it?

**Comment by guihkx** [comment]
@HCY-ASLEEP Can you check if that's still a problem with our new Qt 6 build?

You can download the latest developer AppImage from here: https://github.com/nuttyartist/notes/suites/10381469949/artifacts/512282569
